news/11909.feature.rst

```diff
@@ -0,0 +1 @@
+Most project metadata is now defined statically in the ``pyproject.toml`` file.
```

Comment:
Since this is not user-visible, adding this to the changelog might create confusion.

Reply:
Alternatively, this should be reformulated to make it more explicit that it is pip's own pyproject.toml.

Reply:
I tried to clarify it.

Reply:
FWIW, the reason I decided to make an entry after all is that this could _potentially_ be of interest to people using tools to examine sdist files. It's not an especially prominent change, but it is externally visible. 🤷‍♂️

Reply:
`11909.process.rst` is more appropriate than `feature` in this case.

Reply:
@SpecLad there is still the renaming of the newsfragment to do.

Reply:
Oh, sorry, I forgot to push this. Done now.